### ISSUE TITLE
AutoComplete missing visual cue for invalid form value

### ISSUE THREAD

**Issue by tapopov** [telerik/kendo-angular#997]
## I'm submitting a...

* Bug report 

## Current behavior

An empty AutoComplete component with a `required ` property does not have a visual cue indicating that the value is invalid.
## Expected behavior

A red border around the component is expected.
## Minimal reproduction of the problem with instructions
  1. Open [example](http://plnkr.co/edit/cvd3NYtQ1r4Ie5UBjVP9?p=preview)
  2. Clear the AutoComplete
  3. Click outside to blur

## What is the motivation or use case for changing the behavior?

Consistency and matching user expectations.

## Workaround
A custom CSS rule can be used as a workaround:
```
.k-widget.ng-invalid.ng-dirty {border-color: rgba(243,23,0,.5);}
```

**Comment by ipeshev** [comment]
Fixed in v2.43.3-dev.201710301155

**Comment by gyoshev** [comment]
Released in `@progress/kendo-theme-default@2.44.0`
